.github/workflows/ci.yml

```diff
@@ -60,9 +60,9 @@ jobs:
     runs-on: ubuntu-latest
     needs: [ build-dependencies, linux-validate-format ]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - id: files
-        uses: jitterbit/get-changed-files@v1
+        uses: tj-actions/changed-files@v34.4.3
```

Comment:
Let's stick with [v34](https://github.com/tj-actions/changed-files/releases/tag/v34). I don't expect breaking changes in lesser releases (e.g. removing/renaming of inputs etc.). This way, we could have a benefit of potential fixes.

Reply:
done, thank you